components/reddit/sources/new-comments-by-user/new-comments-by-user.js

```diff
@@ -0,0 +1,95 @@
+const common = require("../../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-by-user",
+  name: "New comments by user",
+  description: "Emits an event each time a user posts a new comment.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    username: { propDefinition: [reddit, "username"] },
+    context: {
+      type: "integer",
+      label: "context",
+      description: "an integer between 2 and 10",
```

Comment:
We should provide a better description, enough so that users can make a decision on what to put here.

We can also add some attributes to sanitise the data:
```js
module.exports = {
  props: {
    context: {
      // Other attributes here...
      min: 2,
      max: 10,
    },
  },
};
```

Reply:
I agree. I went ahead and look for the description of this parameter. Only then I updated the description here in this prop.